docker/init.sh

```diff
@@ -1,14 +1,17 @@
 #!/bin/bash
 
+set -e
+
 echo 'Starting PrairieLearn...'
 
 cd /PrairieLearn
 make -s start-support
-node server.js --migrate-and-exit >/dev/null
 
 if [[ $NODEMON == "true" || DEV == "true" ]]; then
+    make dev-migrate > /dev/null
```

Comment:
```suggestion
    make migrate-dev > /dev/null
```

Reply:
I agree that the syntax of "command verb noun" is good here (like `git clone repo`).

Reply:
It's a bugfix, not a suggestion. The target here does not match the name in the Makefile.

Reply:
Thanks for pointing this out, will fix.

To Matt's point, maybe the terminology is lacking? `-dev` in this context is more of an adjective than a verb here as I explained below.

An alternative way of thinking about this: we could use a `start-dev` command instead of just `dev`, and then we'd have `start : start-dev :: migrate : migrate-dev`.

Reply:
`start-dev` and `migrate-dev` feel natural to me.

Reply:
I can add `start-dev` as an alias for `dev` in another PR if desired?